Code/Engine/RendererVulkan/CommandEncoder/Implementation/CommandEncoderImplVulkan.cpp

```diff
@@ -1195,6 +1207,10 @@ ezResult ezGALCommandEncoderImplVulkan::FlushDeferredStateChanges()
     m_TextureAndSampler.Clear();
     const ezUInt32 uiSets = m_PipelineDesc.m_pCurrentShader->GetSetCount();
     m_DescriptorSets.SetCount(uiSets);
+
+    ezDeque<vk::DescriptorBufferInfo> dynamicUniformBuffers;
```

Comment:
Per frame allocation(s), keep around as member like the others above.

Reply:
Fixed!